gr-uhd/examples/grc/rfnoc_images/x310_image_core.grc

```diff
@@ -0,0 +1,559 @@
+options:
```

Comment:
As a courtesy this is the GRC file:

![x310_image_core](https://github.com/user-attachments/assets/71fed2ce-b6ab-4d46-8afc-3cdfe28a0820)